content/blog/v1-33-released.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Scratch Addons v1.33 is here!
+description: Badges for project creators in comments, a keyboard shortcut for pausing, and more!
+date: 2023-06-09T00:00:00Z
```

Comment:
I don't think this is right.

Reply:
Uh oh. @WorldLanguages We forgot something!

Reply:
I'll fix it.

Reply:
#346